content/400-pulse/200-getting-started.mdx

```diff
@@ -58,6 +59,8 @@ ALTER SYSTEM SET wal_keep_size = 2048;
 
 ### Provider specific configuration
 
+To learn about the database providers that Pulse supports, visit [here](/pulse/faq#what-database-providers-are-supported-with-pulse).
+
 #### Railway
 
 [Railway.app](https://railway.app) offers an excellent [templates feature](https://railway.app/templates). If you wish to quickly start with Pulse, you can use either of two templates:
```

Comment:
I believe the restart button for the Railway template is now under the "Deployments" tab right?

Reply:
That's a great catch @grpatel 💯